### DISCUSSION TITLE
Support Request: Huion HS610 isn't detected by OpenTabletDriver

### DISCUSSION THREAD

**Discussion by Cosmic-Valkyrie** [OpenTabletDriver/OpenTabletDriver#3479] [General]
### What do you need help with?

OpenTabletDriver wont detect my Huion HS610. I looked at the debug section of the console and it says that it's searching for it but it wont detect while my tablet is plugged in

### What have you tried?

I tried restarting my computer and I tried to use WinUSB and I looked through the "My tablet is supported but not detected" sections of the FAQ and and I made sure that I have no other drivers on my system but nothing worked

### Diagnostic Information

[Huion HS610.json](https://github.com/user-attachments/files/17092782/Huion.HS610.json)

### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.
- [X] I am on the latest version of OpenTabletDriver.

**Comment by jamesbt365** [comment]
https://opentabletdriver.net/Wiki/FAQ/Windows#driver-cleanup

**Reply by Cosmic-Valkyrie** [reply]
That Didn't work, do you have any other things I can try?

**Reply by jamesbt365** [reply]
open device manager > view > device by container > find your tablet > open the container, screenshot.

**Reply by Cosmic-Valkyrie** [reply]
![image](https://github.com/user-attachments/assets/aa0e2213-faa3-4c0a-a58d-252946f0caaa)

**Reply by jamesbt365** [reply]
In OpenTabletDriver, tablets > device string reader
9580
100

dump all, save that file and send it here.

**Reply by jamesbt365** [reply]
Nothing, you are dumping *all* indexes.

**Reply by Cosmic-Valkyrie** [reply]
[huion hs610.txt](https://github.com/user-attachments/files/17098244/huion.hs610.txt)

**Reply by jamesbt365** [reply]
Create a folder called `Configurations` next to OpenTabletDriver and place this file in there.
[HS610.json](https://github.com/user-attachments/files/17098331/HS610.json)

**Reply by Cosmic-Valkyrie** [reply]
That's it? like, it works now?

**Reply by jamesbt365** [reply]
Well replug your tablet and if it works it works.

**Reply by Cosmic-Valkyrie** [reply]
thank you!

**Reply by MCUxDaredevil** [reply]
Hi, I am facing the same problem on Linux - Ubuntu 24
I tried Device String reader but it doesn't seem to do anything for me
It keeps asking to replug and press OK

**Comment by ghost** [comment]
The device string reader isn't letting me put in letters. My vendor ID has a letter in it.

**Reply by jamesbt365** [reply]
Hex to base-10.

**Reply by ghost** [reply]
thx

**Comment by Duisternis** [comment]
If someone is facing this on macOS. Trying removing $ from DeviceStrings, apparently MacHidDevice.GetDeviceString gives some trailing garbage after the tablet name.

```json
"DigitizerIdentifiers": [
  {
    ...
    "DeviceStrings": {
      "201": "HUION_T194_\\d{6}$" // change to HUION_T194_\\d{6}
    },
```
